### ISSUE TITLE
Support cleaning parts of XML using CleanerML

### ISSUE THREAD

**Issue by Tobias-B-Besemer** [bleachbit/wishlist#29]
BleachBit should have a XML Cleaner.

Programs found so far that store data in XML files:
- Light Alloy
- Double Commander
- Notepad++

**Comment by Tobias-B-Besemer** [comment]
I did a other sample in the double_commander.xml... -> https://github.com/az0/cleanerml/pull/202

**Comment by az0** [comment]
I agree this would be nice

**Comment by Tobias-B-Besemer** [comment]
I saw that the Python Standard Library includes limited support for XPath expressions...
https://en.wikipedia.org/wiki/XPath#Python

**Comment by az0** [comment]
Yes, that might help. There actually is an old XML cleaner for OpenOffice, but it's not in cleanerml.

**Comment by Tobias-B-Besemer** [comment]
I want to docu a bit the cleaners that need XML-Support for later... (...to come back to them...)

Cleaners that need XML, too:
- filezilla.xml
- OpenOffice.org (See: https://github.com/bleachbit/bleachbit/issues/480)

**Comment by Tobias-B-Besemer** [comment]
I found a other scenario in filezilla.xml...
...here is a syntax that I can imagine...
`<action command="xml" search="file" path="$$profile$$/filezilla.xml" element="FileZilla3/Settings/Setting" attribute="name=Last Server Path"/>`
Explanation: Deletes with a "name=value" as value of "attribute" the content of "<element attribute="value">".
...Example:
`    <FileZilla3 version="3.32.0" platform="windows">`
`        <Settings>`
`            <Setting name="Last Server Path">ftp://ftp.demo.org</Setting>`
-> Deletes "ftp://ftp.demo.org"

**Comment by Tobias-B-Besemer** [comment]
I found a other scenario in filezilla.xml...
...here is a syntax how I can imagine...
`<action command="xml" search="file" path="$$profile$$/filezilla.xml" element="FileZilla3/Settings/Setting" attribute="name=Last Server Path"/>`
Explanation: Deletes with a "name=value" as value of "attribute" the content of "<element attribute="value">".
...Example:
`    <FileZilla3 version="3.32.0" platform="windows">
        <Settings>
            <Setting name="Last Server Path">ftp://ftp.demo.org</Setting>`
-> Deletes "ftp://ftp.demo.org"
